features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -37,7 +37,11 @@
 
     <feature name="openhab-binding-amazondashbutton" description="Amazon Dash Button Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
-        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.amazondashbutton/${project.version}</bundle>
+        <bundle dependency="true">mvn:com.google.guava/guava/21.0</bundle>
+        <bundle dependency="true">mvn:net.java.dev.jna/jna/4.2.1</bundle>
+        <bundle dependency="true">wrap:mvn:org.pcap4j/pcap4j-core/1.6.6</bundle>
+        <bundle dependency="true">wrap:mvn:org.pcap4j/pcap4j-packetfactory-static/1.6.6</bundle>
```

Comment:
After making a local distro build I found out that we need to add the following prerequisite to features in order to be able to use the wrap protocol in features:

`<feature prerequisite="true">wrap</feature>`

We might want to move that to the openhab-runtime-base feature so we don't have to duplicate it in many add-on features.

Also if the bundle name/symbolic name/version are not added to the wrap URL, these wrapped JARs show up very ugly with `bundle:list`:

```
220 │ Active │  80 │ 0                     │ wrap_file__home_wouter_software_openhab_instance_userdata_tmp_kar_openhab-addons-2.5.0-SNAPSHOT_com_google_api_api-common_1.7.0_api-common-1.7.0.jar
221 │ Active │  80 │ 0                     │ wrap_file__home_wouter_software_openhab_instance_userdata_tmp_kar_openhab-addons-2.5.0-SNAPSHOT_com_google_api_gax_1.42.0_gax-1.42.0.jar
222 │ Active │  80 │ 0                     │ wrap_file__home_wouter_software_openhab_instance_userdata_tmp_kar_openhab-addons-2.5.0-SNAPSHOT_com_google_auth_google-auth-library-credentials_0.13.0_google-auth-library-credentials-0.13.0.jar
223 │ Active │  80 │ 0                     │ wrap_file__home_wouter_software_openhab_instance_userdata_tmp_kar_openhab-addons-2.5.0-SNAPSHOT_com_google_auth_google-auth-library-oauth2-http_0.13.0_google-auth-library-oauth2-http-0.13.0.jar
224 │ Active │  80 │ 0                     │ wrap_file__home_wouter_software_openhab_instance_userdata_tmp_kar_openhab-addons-2.5.0-SNAPSHOT_com_google_http-client_google-http-client-jackson2_1.28.0_google-http-client-jackson2-1.28.0.jar
```

Can you add some parameters for these bundle manifest entries similar to #5219?

Reply:
Sure. I‘ll also check if we can get rid of guava easily.